Documentation/Index.rst

```diff
@@ -45,6 +45,7 @@ continuous integration.
    Introduction
    PHPVersionSupport
    TestingFramework
```

Comment:
You'll also need to remove this line. (With this line not removed, building the documentation fails.)

To catch build errors early, I strongly recommend building the documentation locally before you push a new version:

```bash
./Build/Scripts/runTests.sh -s docsGenerate
```

Reply:
Just removed TestingFramework from Table of Contents. Now the documentation builds successfully.